sklearn/linear_model/tests/test_coordinate_descent.py

```diff
@@ -888,3 +888,12 @@ def fit(self, X, y):
     clf = LassoCV(precompute=precompute)
     clf.fit(X, y)
     assert calls > 0
+
+
+def test_multi_task_lasso_cv_dtype():
+    n_samples, n_features = 10, 3
+    rng = np.random.RandomState(42)
+    X = rng.binomial(1, .5, size=(n_samples, n_features))
```

Comment:
It's not necessarily clear to me while reading this that it would return a dtype int. Maybe worth, explicitly casting to int for readability,
```suggestion
    X = rng.binomial(1, .5, size=(n_samples, n_features)).astype(np.int)
```

Reply:
done below.